.gitignore

```diff
@@ -7,4 +7,5 @@ okta-oidc-tck
 samples-nodejs-express-4
 .DS_Store
 .vscode
-.samples.config.js
\ No newline at end of file
+.samples.config.js
+.samples.config.js--
```

Comment:
I too had this file generated. Maybe we should look at the `setup.sh` script instead of attempting to ignore it?

Reply:
If you replace `sed -i --` with `sed -i ''` it will resolve this. See https://stackoverflow.com/a/7573438/7059401

Reply:
@jmelberg-okta - I have tried it but it works only on linux machines. On Macs, it threw an error if I remember correctly.

Reply:
Odd.. `sed -i ''` works great on my machine.

![Image](https://vangogh.teespring.com/v3/image/BITOunwdicI0KK_7_jCM5F-Yiwo/480/560.jpg)

Reply:
Yes, I tried it locally and it worked. Looks like it failed on travis. Let me try on travis and check.